python/pyspark/sql/protobuf/functions.py

```diff
@@ -49,7 +49,10 @@ def from_protobuf(
     data : :class:`~pyspark.sql.Column` or str
         the binary column.
     messageName: str
-        the protobuf message name to look for in descriptor file.
+        the protobuf message name to look for in descriptor file. Or
+        The Protobuf class name. E.g. <code>org.spark.examples.protobuf.ExampleEvent</code>,
```

Comment:
```suggestion
        The Protobuf class name. E.g. ``org.spark.examples.protobuf.ExampleEvent``,
```

Reply:
@HyukjinKwon thanks for the quick review made changes.